public_officials/static/public_officials/js/legislator_detail.js

```diff
@@ -7,6 +7,7 @@ var getIndustryContributors = function(id){
       );
     } else {
       industryDonationsChart(data)
+      totalDonationsChart(data)
```

Comment:
There are a few repeats of the same enumerable, does it make the most sense to make a function for each of the different things happening for each row? Should some be combined while other stay separate?